src/AssemblerOptions.cpp

```diff
@@ -594,6 +601,7 @@ void AssemblerOptions::ReadsOptions::write(ostream& s) const
 {
     s << "[Reads]\n";
     s << "minReadLength = " << minReadLength << "\n";
+    s << "desiredCoverage = " << desiredCoverage << "\n";
```

Comment:
We should write `desiredCoverageString` here, to exactly reflect what we got from the user when running the assembly. I will still merge the PR - you can make this change later.